main/libwebp/APKBUILD

```diff
@@ -18,7 +18,6 @@ prepare() {
 }
 
 build() {
-	cd "$builddir"
```

Comment:
FWIW, removing those `cd "$builddir"` lines may require more work if/when backporting to stable branches, so in general, it would be nice to keep them for a while. At least in `main` repo.